doc/changelog.rst

```diff
@@ -56,20 +56,24 @@ MJX
 11. Added support for ``eq_active``. Fixes :github:issue:`2173`.
 12. Added ray intersection with ellipsoid.
 
+Python bindings
+^^^^^^^^^^^^^^^
+13. Removed ``nroll`` argument from rollout because its value can always be implied
```

Comment:
This is the wrong section? Should be at the top of the file "Upcoming version (not yet released)"

Reply:
Whoops, my apologies.